homework-2-html/exercitiu.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width" />    
+        <title>Pizza Order</title>
+    </head>
+    <body>
+        <fieldset>
+            <legend>Pizza Order</legend>
+        
+        <form action="pizza.html" method="get">
+                <label for="name:"></label>
+                <input text="text" id="name:" name="name" required minlength="3" maxlength="10" size="10" placeholder="Name" >
+                <br>
+                <br>
+                <label for="email"></label>
+                <input text="email" id="email" pattern=".+@\.com" size="10" required placeholder="email" >
```

Comment:
pentru type="email" nu ai mai avea nevoie de pattern de verificare

```suggestion
                <input type="email" id="email" size="10" required placeholder="email" >
```